frontend/src/pages/org/crawl-configs-list.ts

```diff
@@ -109,9 +109,23 @@ export class CrawlTemplatesList extends LiteElement {
 
   render() {
     return html`
-      <div class="sticky z-10 mb-2 top-0 py-2 bg-neutral-0">
+    <header class="contents">
```

Comment:
Seeing an indentation issue, could you check whether prettier is running on pre-commit? I also pushed an [editorconfig to main](https://github.com/webrecorder/browsertrix-cloud/commit/5b2e101d20a1bcdb9f6ff307f95a190ec5bd3840) if you want to try that out.

Reply:
Should be fixed, IDK why that's not running automatically(?) 🤔

Something to investigate later...